src/CitationProvider.ts

```diff
@@ -14,36 +14,40 @@
  * limitations under the License.
  */
 
-import locales from '@manuscripts/data/dist/csl/locales/locales.json'
 import { BibliographyItem } from '@manuscripts/json-schema'
 import CiteProc, { Citation as CiteProcCitation } from 'citeproc'
 
 import { variableWrapper } from './citeproc-variable-wrapper'
 import { convertBibliographyItemToCSL } from './csl-converter'
+import defaultLocale from './defaultLocale'
 
 interface Props {
   getLibraryItem: (id: string) => BibliographyItem | undefined
   citationStyle: string
-  lang?: string // Default en-GB
+  lang?: string // Default en-US
```

Comment:
Is this needed anymore?

Reply:
will use it in [citeproc](https://github.com/Atypon-OpenSource/manuscripts-library/blob/master/src/CitationProvider.ts#L50)

Reply:
Yes, but does the value matter in our case? citeproc will just use `retrieveLocale` to get the locale config, so can we just pass a static value ("en" for example) to citeproc?

Reply:
updated